go/vt/sqlparser/parse_test.go

```diff
@@ -1156,6 +1156,9 @@ var (
 	}, {
 		input:  "CREATE TABLE aipk (id INT AUTO_INCREMENT PRIMARY KEY)",
 		output: "create table aipk (\n\tid INT auto_increment primary key\n)",
+	}, {
+		input:  "create table foo (f timestamp null not null , g timestamp not null null)",
```

Comment:
this I don't understand. what does it mean to say both null and not null about the column? why are we picking one of them here instead of failing?

Reply:
added a comment for it